src/librustdoc/clean/auto_trait.rs

```diff
@@ -33,9 +33,10 @@ impl<'a, 'tcx> AutoTraitFinder<'a, 'tcx> {
     }
 
     // FIXME(eddyb) figure out a better way to pass information about
-    // parametrization of `ty` than `param_env_def_id`.
-    crate fn get_auto_trait_impls(&self, ty: Ty<'tcx>, param_env_def_id: DefId) -> Vec<Item> {
-        let param_env = self.cx.tcx.param_env(param_env_def_id);
+    // parametrization of `ty` than `item_env_def_id`.
```

Comment:
I don't see anything named `item_env_def_id`. Do you mean:

```suggestion
    // parametrization of `ty` than `item_def_id`.
```

?

Reply:
I removed the comment altogether, I don't think it makes sense now that `ty` isn't passed as an argument.